index.css

```diff
@@ -38,3 +38,9 @@ nav {
 .page-selector > *:not([selected]) {
   display: none !important;
 }
+
+@media (max-width: 600px) {
```

Comment:
This should actually be `var(--content-width)`, but variables are not allowed in media queries. Maybe just always add the margin?